.github/workflows/push.yml

```diff
@@ -18,13 +18,7 @@ concurrency:
 
 env:
   # Not needed in CI, should make things a bit faster
-  CARGO_INCREMENTAL: 0
   CARGO_TERM_COLOR: always
-  # Build smaller artifacts to avoid running out of space in CI
-  # TODO: Try to remove once https://github.com/paritytech/substrate/issues/11538 is resolved
-  RUSTFLAGS: -C strip=symbols -C opt-level=s
```

Comment:
These were not useless, they were intentionally added to decrease size of build artifacts, otherwise we can run out of disk space on Windows (maybe not an issue for SDK, but certainly an issue for monorepo where we're building many huge executables for tests.

Reply:
Why release for tests? It makes compilation significantly slower and likely not worth it if you want to speed things up.

Reply:
For tests which involve farming lots of blocks CI is still very slow. Even in `--release` mode it can take 10-30 mins (without compilcation). But compilation itself isn't that slower though

Reply:
Well, do you really need to farm a lot of blocks though? Also block verification is very slow right now, we'll make it much faster soon (not necessarily block production though).

Reply:
Sorry, I meant plotting + farming blocks is very slow. It takes more than 1-2 minutes on my machine for 4 blocks and it takes 4-5 minutes for CI for release build.

Also I encountered that solution verification might be so slow that it takes longer to verify it than a slot time, therefore CI couldn't produce blocks and test failed.

Reply:
Hm... how many sectors is it plotting? Does it set initial solution range to the maximum? Multiple minutes seems way too long.

Reply:
> how many sectors is it plotting

I currently set plot size to be 1G. I don't remember solution range, but I believe it is the same as in monorepo (I think initial solution range is not customizable in chainspec). What plot size would you suggest?